libkbfs/interfaces.go

```diff
@@ -1502,11 +1503,15 @@ type BareRootMetadata interface {
 	// folder.  This is only expected to be set for folder resets.
 	IsFinal() bool
 	// IsWriter returns whether or not the user+device is an authorized writer.
-	IsWriter(user keybase1.UID, deviceKID keybase1.KID) bool
+	// Note "wkb" is expected to be null for pre-v3 metadata.
+	IsWriter(user keybase1.UID, deviceKID keybase1.KID, wkb *TLFWriterKeyBundleV2) bool
```

Comment:
It might be better to have an interface for the data that's external to the BRMD itself. Perhaps name it `ExternalMetadata`, and then have all these functions take a parameter of that type. Then v2 can ignore it / assert that it's nil, and v3 can cast it to a struct with the WKB/RKB. What do you think?